config/dev.exs

```diff
@@ -61,7 +61,7 @@ config :fz_wall,
 
 config :fz_vpn,
   wg_path: "wg",
```

Comment:
Do we still need wg_path?

Reply:
I don't believe so, I'll remove these from the configs.